Kernel/style.m

```diff
@@ -1,3 +1,5 @@
+(* ::Package:: *)
```

Comment:
No, I think that just got added automatically. I'm happy to remove if it's inconsistent.

Reply:
In this case, I think it's better to remove it from here since this file was not originally created by the front end, and does not have other related stuff (like sections, etc.)